pkg/reconciler/managed/reconciler.go

```diff
@@ -797,6 +799,7 @@ func (r *Reconciler) Reconcile(_ context.Context, req reconcile.Request) (reconc
 
 	if observation.Diff != "" {
 		log.Debug("External resource differs from desired state", "diff", observation.Diff)
+		record.Event(managed, event.Normal(reasonNeedReconcile, observation.Diff))
```

Comment:
```suggestion
		record.Event(managed, event.Normal(reasonNeedReconcile, "Diff:\n" + observation.Diff))
```

What do you think about this change? I figure since the diff will always be multi-line it might read more clearly if it starts on its own line.